update developer docs for 1proposal flow (#1814)
dfinity/portal: docs/developer-docs/integrations/sns/launching/index.md

# Launching an SNS

This Section describes the SNS launch process in detail.

> It starts with a detailed description of all Stages of an SNS launch.

Reviewer: ```suggestion
It starts with a detailed description of all stages of an SNS launch.
```
?

Author: yes. applying.

> Then, it explains the practical commands & actions to go through SNS launch, in terms of concrete tools and commands that are needed to take a project through all of these stages and launch an SNS for it.

Author: I slightly rephrased as I think the "terms" is not something "that is needed". 
Please lt me know if you think more adjustments are needed

> In August 2023, the NNS has decided to release a new version of the SNS with a new, simplified launch process. In particular, the new launch process only requires one single NNS proposal that will, if adopted, trigger the creation of the SNS canisters as well as starting and finishing the decentralization swap. Since the new flow is considerably simpler both for developers who want to hand over their dapp to the IC and for voters/users, it is recommended to use the new flow.

Reviewer: ```suggestion
In August 2023, the NNS voted to release a new version of the SNS with a new, simplified launch process. In particular, the new launch process only requires one single NNS proposal that will, if adopted, trigger the creation of the SNS canisters as well as starting and finishing the decentralization swap. Since the new flow is considerably simpler both for developers who want to hand over their dapp to the IC and for voters/users, it is recommended to use the new flow.
```

As some dapp projects might already have started with planning and testing the SNS launch process, the old legacy flow is still supported. Note that the legacy flow might be deprecated going forward.

Therefore, the developer documentation currently includes pages to describe both the legacy and the new flow.

> The pages [Stages of an SNS launch](./launch-summary-1proposal.md) and [Commands & actions to go through SNS launch](./launch-steps-1proposal.md) define the new, recommended 1-proposal launch flow and how to go through it.

Author: I would not rename the second page. Even though the name of the page is still "steps" in the files, I wanted to rename the page where it is visible to the users.
Reason: I think it is unclear what's teh difference between "stages" and "steps". Rather, one page should describe the stages and the other one "how to go through these stages".

> Therefore, the developer documentation currently includes pages to describe both the legacy and the new flow.
> The pages [Stages of an SNS launch](./launch-summary-1proposal.md) and [Commands & actions to go through SNS launch](./launch-steps-1proposal.md) define the new, recommended 1-proposal launch flow and how to go through it.
> The pages [Stages of an SNS launch](./launch-summary.md) and [Commands & actions to go through SNS launch](./launch-steps.md) define the old, legacy flow that might be deprecated going forward.

Reviewer: ```suggestion
Therefore, the developer documentation currently includes pages to describe both the legacy and the new flow.
The pages [Stages of an SNS launch](./launch-summary-1proposal.md) and [Commands & actions to go through SNS launch](./launch-steps-1proposal.md) define the new, recommended 1-proposal launch flow and how to go through it.
The pages [Stages of an SNS launch - legacy flow](./launch-summary.md) and [Commands & actions to go through SNS launch - legacy flow](./launch-steps.md) define the old, legacy flow that might be deprecated going forward.
```

Author: good idea. will do
